ansible/roles/ooni-backend/tasks/populate_bouncer_base.yml

```diff
@@ -15,13 +15,32 @@
     - "{{ bind_collector_hsdir }}"
     - "{{ bind_web_connectivity_hsdir }}"
 
-- name: Template "{{ bind_bouncer_file  }}"
+- name: Template "{{ bind_bouncer_file }}" with bouncer_base.yaml.j2
   template: >
     src="bouncer_base.yaml.j2"
     dest="{{ bind_bouncer_file }}"
     backup="yes"
+    unsafe_writes="true"
+  when: ( oonibackend_bundle is defined and oonibackend_bundle == True )
+  register: bouncer_config_file
 
-- name: Fetch backend related config files and HS keys
+- name: Template "{{ bind_bouncer_file }}" with "{{ bouncer_base_template }}"
+  template: >
+    src="{{ bouncer_base_template }}"
+    dest="{{ bind_bouncer_file }}"
+    backup="yes"
+    unsafe_writes="true"
+  when: ( oonibackend_bundle is undefined or oonibackend_bundle == False )
```

Comment:
> is there any case where the conditional for oonibackend_bundle variable in #79 (diff) fails to evaluate when oonibackend_bundle is unset or set to: Yes/yes, No/no, True/true, False/false ?

Yes, in old version of jinja2, where it was incorrect to say "False" instead of "false", it seems like recent versions don't behave that way anymore, but it's still incorrect to use "False" instead of "false": http://jinja.pocoo.org/docs/dev/templates/#expressions.

> If not I will prefer to use the same conditional syntax as in most (if not all) ansible roles/playbooks in this repository.

I don't see this pattern used anywhere else in the repo actually (the check for true false is only used here), while the is undefined check is used in only 7 places:
```json
$ grep -rn 'is undefined' ansible | wc -l
       7
```

On the other hand the casting to a default value is used in many more places:
```
$ grep -rn 'default(' ansible | wc -l
      24
```

I would change the 7 lines above to use this better pattern.